docs/userGuide/syntax/dates.mbdf

```diff
@@ -0,0 +1,98 @@
+## Dates
+
+{% macro njcode(raw) %}<code>{<a/>{ {{ raw }} }}</code>{% endmacro %}
+{% macro njblock(raw) %}<code>{<a/>% {{ raw }} %}</code>{% endmacro %}
+
+**Markbind supports date formatting and simple calculations** as a Nunjucks filter.
```

Comment:
Might be nice to link to https://mozilla.github.io/nunjucks/templating.html#filters